Formula/hiptest-publisher.rb

```diff
@@ -0,0 +1,97 @@
+class HiptestPublisher < Formula
+  desc "Publisher for HipTest projects"
+  homepage "https://github.com/hiptest/hiptest-publisher"
+  url "https://github.com/hiptest/hiptest-publisher/archive/v2.0.0.tar.gz"
+  sha256 "ec2e89637655935616e7f427778c4dbe205891ea0f58e9b30c7130532a48dcfe"
+  head "https://github.com/hiptest/hiptest-publisher.git"
+
+  depends_on "ruby"
+
+  resource "colorize" do
```

Comment:
Is there no lock file to lock all dependencies to the correct version?

Reply:
Yes, there is a Gemfile.lock in the repository. I guess I should use `bundle install` to install all dependencies ?

Reply:
Please do

Reply:
sure :+1:

Reply:
Done.

Reply:
Done.

Reply:
I don't understand why tests are failed. If you have any ideas. Thanks.